enhancements/baremetal/enable-baremetal-on-other-platforms.md

```diff
@@ -0,0 +1,197 @@
+
+2. documentation will need to be added to the centralized host management documentation
+explaining how to create a Provisioning CR. Current documentation is here:
+https://github.com/openshift/assisted-service/blob/8880093ef5ce041d4c1951ffd5ea1096991ec3ee/docs/user-guide/assisted-service-on-openshift.md#configure-bare-metal-operator
+
+### User Stories
+
+#### Story 1 - Current IPI baremetal platform use case
+
+No change.
+
+#### Story 2 - centralized host management use case
+
+As a user of a hub cluster that performs central infrastructure management, and
+optionally zero-touch provisioning, I need to provision hosts using the k8s-native
+API (Baremetal Hosts CR) even when the hub cluster has a platform of None, OpenStack, or vSphere.
+
+
+### Risks and Mitigations
+
+There is concern that *random* customers will use this feature out of context
+and create support burden. This is why we have not suggested enabling CBO on
+all platforms and with full feature set. However it is still a potential issue.
+
+## Design Details
+
+### Open Questions [optional]
+
+1. Where will the feature be e2e tested?
+
+### Test Plan
+
+#### Unit Testing
+
+We will add unit tests to confirm that cluster-baremetal-operator:
+* is enabled on the required platforms.
+* will restrict functionality on these platforms to ProvisioningNetork=Disabled.
+
+#### Functional Testing
+
+An e2e test will be written in the Assisted Installer CI that will:
+1. create one of the platforms above (SNO Platform=None might be the easiest) with Assisted Service.
+2. confirm that CBO is enabled
+3. create a Provisioning CR and confirm that BMO is running
+4. provision a baremetal cluster
+
+QE will validate the remaining platforms that are supported to reduce the load
+on CI.
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+#### Tech Preview -> GA
+
+The feature will go to GA without tech preview
+
+#### Removing a deprecated feature
+
+### Upgrade / Downgrade Strategy
+
+cluster-baremetal-operator will upgrade as it currently does, this is only a
+minor change in functionality.
+
+On the platforms (None, OpenStack and vSphere) where the operator was in a disabled
+state, after been upgraded it will move into an enabled state. However in all but
+centralized host management use cases nothing will change as there is no Provisioning CR.
+
+### Version Skew Strategy
+
+None required as this is not dependant on other components.
+
+## Implementation History
+
+This PR is the current WIP implementation: https://github.com/openshift/cluster-baremetal-operator/pull/189
+
+## Drawbacks
+
+There is concern that *random* customers will use this feature out of context
+and create support burden.
+
+## Alternatives
+
+Customers can instead create a dedicated baremetal cluster to use as the hub
+cluster.
+
+Another alternative is to additionally distribute baremetal-operator as an optional
+operator with OLM. It's probably not a good idea, but worth enumerating as an alternative.
```

Comment:
"It's probably not a good idea, but worth enumerating as an alternative." can be removed.